tests/components/homekit_controller/specific_devices/test_hue_bridge.py

```diff
@@ -19,11 +16,11 @@
     battery = entity_registry.async_get(battery_id)
     assert battery.unique_id == "homekit-6623462389072572-644245094400"
 
-    battery_helper = Helper(
-        hass, "sensor.hue_dimmer_switch_battery", pairing, accessories[0], config_entry
-    )
+    battery_helper = Helper(hass, "sensor.hue_dimmer_switch_battery", pairing,
+                            accessories[0], config_entry)
     battery_state = await battery_helper.poll_and_get_state()
-    assert battery_state.attributes["friendly_name"] == "Hue dimmer switch Battery"
+    assert battery_state.attributes[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)